modules/tensor_mechanics/tests/finite_strain_elastic/equation_of_state.i

```diff
@@ -0,0 +1,135 @@
+# Isotropic compression test on one element
+# for a linear elastic material with Murnaghan equation of state
+# expected stress_zz is: P = (K0 / n) * J^{-n}
+# J is the Jacobian: J = detF = v / v0, F = deformation gradient
+# v and v0 are the specific volumes: deformed and reference
+# Austin et al. JOURNAL OF APPLIED PHYSICS 117, 185902 (2015)
```

Comment:
I have reported the analytical solution for the component stress_zz, which is equal to the pressure P, given by the equation of state. The value at the final timestep is equal to the analytical value, I have reported this value in the comment